css/style.css

```diff
@@ -0,0 +1,348 @@
+  justify-content: space-between;
+  padding: 0 24px;
+}
+
+.my-logo {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  animation: logoAnimate 3s alternate-reverse 3s 2;
+}
+
+.hamburger {
+  border: none;
+  background: white;
+  color: #6070ff;
+  cursor: pointer;
+}
+
+#main-page {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #fff;
+  background-image: url(/images/backgound_image.svg);
+  background-repeat: no-repeat;
+  background-position: -20px 80px;
+  margin-bottom: 4rem;
+  padding-top: 2.2rem;
+}
+
+.cover-socials a {
+  text-decoration: none;
+  font-size: 22px;
+  color: #505f79;
+}
+
+.socials a {
+  text-decoration: none;
+  font-size: 22px;
+  color: #505f79;
+}
+
+@keyframes logoAnimate {
+  from { transform: rotate(0) scale(1); }
+  to { transform: rotate(360deg) scale(1.5); }
+}
+
+a:link {
+  text-decoration: none;
+}
+
+.navbar-menu {
+  display: none;
+}
+
+#navbar .navbar-menu a:hover {
+  color: #396df2;
+  font-weight: 700;
+  cursor: pointer;
+}
+
+.headline {
+  padding: 0 24px;
+  margin-top: 5rem;
+  margin-bottom: 3rem;
+}
+
+.title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.content {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.cover-links {
+  margin-bottom: 10px;
+  color: #7f8cff;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.cover-socials {
+  display: flex;
+  list-style: none;
+  justify-content: flex-start;
+  gap: 10px;
+  padding: 0;
+}
+
+/* ///////// Projects ///////// */
+
+.work-page-left,
+.work-page-right {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: 8px;
+  margin: 0 1rem 4rem 1rem;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.work-title {
+  width: 295px;
+  height: 36px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 6px 0;
+}
+
+.clients {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+}
+
+h4 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.img-clients {
+  padding: 5px;
+}
+
+.highlights {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 4px;
+}
+
+.project-detail {
+  width: 100px;
+  height: 40px;
+  border-width: 1px;
+  border-color: #396df2;
+  border-radius: 0.5em;
+  cursor: pointer;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.04em;
+  color: #396df2;
+  background: #fff;
+  margin-bottom: 10px;
+}
+
+.project-detail:hover {
+  background: #396df2;
```

Comment:
Please make sure your color matches the Figma design. The hover color for this button is  #6070FF. Also please add the "Pressed" state for the buttons. Check the related area on the Figma template.↓
<img width="1091" alt="Screenshot 2023-01-13 at 12 15 47 AM" src="https://user-images.githubusercontent.com/25887253/212183447-a300aecb-2173-4498-9160-1be65dc529a6.png">

Here is the project requirements:
<img width="1091" alt="Screenshot 2023-01-13 at 12 19 54 AM" src="https://user-images.githubusercontent.com/25887253/212183577-43b76db7-3097-4865-bec9-128d81b6b373.png">